h2o-core/src/main/java/water/DKVManager.java

```diff
@@ -111,10 +112,17 @@ protected void setupLocal() {
         }
         if(value.isNull()) continue;
 
-        if (value.isFrame()) {
-          ((Frame) value.get()).retain(removalFutures, retainedKeys);
-        } else if (value.isModel()) {
-          ((Model)value.get()).remove(removalFutures);
+        try {
+          TypeMap.getTheFreezableOrThrow(value.type());
```

Comment:
It declares ClassNotFoundException and makes the catch block possible to compile. The exception itself may occur in the `value.isFrame()` or `value.isModel` calls below as well.